README.md

```diff
@@ -91,11 +91,10 @@ The `:nonbreaking` variant lets us release patch updates quickly in cases where
 
 There are several commands that can be useful in specific scenarios:
 
--   `yarn build:clear-cache` to remove previously created artifacts of the `tsc build` process.
--   `yarn spectrum-vars` to ensure that theme files are up-to-date.
```

Comment:
This command is no longer called directly so I removed it from the docs. All other updates are whitespace fixes from prettier.